src/prpy/controllers/rewd_controllers.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+from ros_control_client_py import FollowJointTrajectoryClient, TrajectoryExecutionFailed
+from ros_control_client_py.util import or_to_ros_trajectory
+
+
+class OrController(object):
+    """Dummy/empty OpenRAVE controller class"""
+
+    def GetControlDOFIndices(self):
+        pass
+
+    def GetNamespace(self):
+        return self.namespace
+
+    def GetRobot(self):
+        return self.robot
+
+    def Reset(self):
+        pass
+
+    def SetDesired(self, positions):
+        pass
+
+    def SetPath(self, traj):
+        pass
+
+    def SimulationStep(self, dt):
+        pass
+
+    def IsDone(self):
+        pass
+
+    def GetTime(self):
+        pass
+    
+    def GetVelocity(self):
+        pass
+
+    def GetTorque(self):
+        pass
+
```

Comment:
Many of these functions return `None` (i.e. `pass`), even though the OpenRAVE equivalent returns a value. These should either: (1) throw a `NotImplementedError` or (2) return a placeholder value of the correct type and size.